share/revng/pipelines/yield/assembly.yml

```diff
@@ -0,0 +1,13 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+From:            Isolate
+Containers:
+ - Name:            assembly.html-body.yml
```

Comment:
```suggestion
 - Name:            assembly.html.yml
```

Reply:
Done: https://github.com/revng/revng/pull/173/commits/f169cc5244b07ecc15e525061a3899834d29af8f and https://github.com/revng/revng/pull/173/commits/9895ae429cd543d4e21eb482db80f4aebc64358f.